controllers/topics-controller.js

```diff
@@ -0,0 +1,18 @@
+//REQUIRES
+const { selectTopics } = require("../models/topics-model");
+
+//FUNCTIONS
+function getTopics(req, res, next) {
+  selectTopics()
+    .then((topics) => {
+      if (topics.length > 0) {
```

Comment:
A better way to handle this would be to check for the appropriate 404 code via PSQL if you do want to handle an error at this point. You are missing an error test, so either remove the error handling at this stage, or test it

Reply:
There is also no else here. It's implied by the if, but if the array is indeed zero, nothing would happen. In fact, we would likely get `undefined`.